### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12997,7 +12997,10 @@ siezed->seized, sized,
 siezing->seizing, sizing,
 siezure->seizure
 siezures->seizures
+sift->shift
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: sift = to use a sieve

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -13030,8 +13033,14 @@ signto->sign to
 signul->signal
 signular->singular
 signularity->singularity
+siguret->cigarette
+silabus->syllabus
 silentely->silently
 silenty->silently
+sill->still
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: exists

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

**Pull request closed without merging**